resources/windows/wix/customactions.wxs.erb

```diff
@@ -278,20 +278,15 @@ If (fso.DirectoryExists(pxpAgentDirectory)) Then
 
   ExecuteCommand comspec & " /C " & icacls & "  """ & pxpAgentDirectory & "\*"" /reset /T /C 2>&1"
 End If
+
+Dim mcoDirectory : mcoDirectory = appDataPath & "\PuppetLabs\mcollective"
+
+If (fso.DirectoryExists(mcoDirectory)) Then
+  WScript.Stdout.WriteLine("Found directory : " & mcoDirectory)
```

Comment:
nit: Debug message should probably be `"Resetting directory : " & mcoDirectory`.  Same with the rest

Reply:
FYI - this changes a little bit in the follow-on PR because it's not just existence being checked, but that's ok.